postgresql/config/postgresql.conf

```diff
@@ -17,7 +17,7 @@ datestyle = 'iso, mdy'
 
 default_text_search_config = 'pg_catalog.english'
 
-data_directory = '{{pkg.svc_data_path}}'
+data_directory = '{{pkg.svc_data_path}}/pgdata'
```

Comment:
While this will work, if anyone automatically updates their `core/postgresql*` package, the data dir will have moved. Something we'll want to think on, but despite that, I like this change.

Reply:
Nevermind, I missed the `init` hook fix